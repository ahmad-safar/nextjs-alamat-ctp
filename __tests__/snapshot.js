import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../pages/index'

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Index data={[]} />).toJSON()
  expect(tree).toMatchSnapshot()
})
