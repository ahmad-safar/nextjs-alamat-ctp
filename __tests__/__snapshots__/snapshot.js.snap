// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="flex flex-col h-screen"
>
  <nav
    className="bg-gray-800"
  >
    <div
      className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8"
    >
      <div
        className="relative flex items-center justify-between h-16"
      >
        <div
          className="absolute inset-y-0 left-0 flex items-center sm:hidden"
        >
          <button
            aria-controls="mobile-menu"
            aria-expanded="false"
            className="inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
            onClick={[Function]}
            type="button"
          >
            <span
              className="sr-only"
            >
              Open main menu
            </span>
            <svg
              aria-hidden="true"
              className="block h-6 w-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 6h16M4 12h16M4 18h16"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
            <svg
              aria-hidden="true"
              className="hidden h-6 w-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6 18L18 6M6 6l12 12"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
          </button>
        </div>
        <div
          className="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start"
        >
          <a
            className="flex-shrink-0 flex items-center"
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="Workflow"
              className="block lg:hidden h-8 w-auto"
              src="/logo.svg"
            />
            <img
              alt="Workflow"
              className="hidden lg:block h-8 w-auto"
              src="/logo.svg"
            />
            <div
              className="text-white px-3 py-2 rounded-md text-sm font-medium"
            >
              Alamat CTP & TAP
            </div>
          </a>
          <div
            className="hidden sm:block sm:ml-6"
          >
            <div
              className="flex space-x-4"
            >
              <a
                className="bg-gray-900 text-white block px-3 py-2 rounded-md text-base font-medium"
                href="/"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                Home
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </nav>
  <main
    className="mb-auto"
  >
    <div
      className="max-w-7xl mx-auto py-6 sm:px-6 lg:px-8"
    >
      <div
        className="px-4 py-6 sm:px-0"
      >
        <div
          className="shadow border-gray-200 rounded-lg px-4 py-6"
        >
          <div
            className="col-span-6 sm:col-span-3"
          >
            <label
              className="block text-sm font-medium text-gray-700"
              htmlFor="country"
            >
              Kecamatan
            </label>
            <select
              autoComplete="country"
              className="mt-1 block w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
              id="country"
              name="country"
              onChange={[Function]}
            >
              <option>
                Pilih Kecamatan
              </option>
            </select>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer
    className="bg-gray-800 text-white"
  >
    <div
      className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8"
    >
      <div
        className="flex items-center justify-between sm:justify-start h-16"
      >
        <div
          className="mr-1"
        >
          Â© 2021 
          <a
            className="ml-1 underline"
            href="https://www.instagram.com/tselkalimantan/"
            target="_blank"
          >
            @tselkalimantan
          </a>
          , Inc.
        </div>
        <div>
          Build by
          <a
            className="ml-1 underline"
            href="https://www.facebook.com/ahmadsafar.1996/"
            target="_blank"
          >
            Ahmad Safar
          </a>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
